pragma/tests/client_test.py

```diff
@@ -246,7 +250,12 @@ async def test_client_oracle_mixin_spot(pragma_client: PragmaClient):
         ETH_PAIR, 100, timestamp + 450, SOURCE_1, publisher_name, volume=10
     )
     try:
-        invocations = await pragma_client.publish_many([spot_entry_future])
+        invocations = await pragma_client.publish_many(
+            [spot_entry_future],
+            l1_resource_bounds=ResourceBounds(
+                max_price_per_unit=500 * 10**9, max_amount=10**7
```

Comment:
explain these values

Reply:
These are arbitrary values. In fact, using `auto_estimate=True` here leads to failure, because the flag auto_estimate full trigger the estimate_fee function call that will revert directly because the simulation fails. By putting l1_resource_bounds to an arbitrary high set value, we avoid the estimate_fee call and the function can indeed fail as expected during the publish_many and not before